catalyst/contrib/nn/criterion/contrastive.py

```diff
@@ -135,8 +135,77 @@ def forward(self, embeddings_pred, embeddings_true) -> torch.Tensor:
         return loss
 
 
+class BarlowTwinsLoss(nn.Module):
+    """The Contrastive embedding loss.
+
+    It has been proposed in `Barlow Twins:
+    Self-Supervised Learning via Redundancy Reduction`_.
+
+    Example:
+        >>> import torch
+        >>> from torch.nn import functional as F
+        >>> from catalyst.contrib.nn import BarlowTwinsLoss
+        >>>
+        >>> embeddings_left = torch.tensor([[1., 0.0],[0.0, 1.0]], requires_grad=True)
+        >>> embeddings_right = torch.tensor([[1., 0.0],[0.0, 1.0]], requires_grad=True)
+        >>> criterion = BarlowTwinsLoss(lmbda = 1)
+        >>> criterion_val = criterion(embeddings_left, embeddings_right).item()
+        >>> abs(1 - criterion_val) < 1e-6
+        True
```

Comment:
could you please use `.. code-block:: python`? 
like https://github.com/catalyst-team/catalyst/blob/master/catalyst/metrics/functional/_f1_score.py#L33#L48
this format is much better for online documentation

Reply:
Yes, sure.

Reply:
[Done](https://github.com/catalyst-team/catalyst/pull/1259/commits/e064ca3698148a7264150b36412b8a48ba774dd5)